autopilot/alerts.go

```diff
@@ -73,23 +73,17 @@ func newAccountRefillAlert(id rhpv3.Account, contract api.ContractMetadata, err
 	}
 }
 
-func newContractPruningFailedAlert(hk types.PublicKey, version string, fcid types.FileContractID, err error) *alerts.Alert {
-	data := map[string]interface{}{"error": err.Error()}
-	if hk != (types.PublicKey{}) {
-		data["hostKey"] = hk.String()
-	}
-	if version != "" {
-		data["hostVersion"] = version
-	}
-	if fcid != (types.FileContractID{}) {
-		data["contractID"] = fcid.String()
-	}
-
-	return &alerts.Alert{
-		ID:        alerts.IDForContract(alertPruningID, fcid),
-		Severity:  alerts.SeverityWarning,
-		Message:   "Contract pruning failed",
-		Data:      data,
+func newContractPruningFailedAlert(hk types.PublicKey, version string, fcid types.FileContractID, err error) alerts.Alert {
+	return alerts.Alert{
+		ID:       alerts.IDForContract(alertPruningID, fcid),
+		Severity: alerts.SeverityWarning,
+		Message:  "Contract pruning failed",
+		Data: map[string]interface{}{
+			"contractID":  fcid.String(),
+			"error":       err.Error(),
+			"hostKey":     hk.String(),
+			"hostVersion": version,
```

Comment:
If we got easy access to the host's `Release` field we could add that here as well.